File: BooksStorage/DTOs/BookReadDTO.cs
using System.ComponentModel.DataAnnotations;

namespace BooksStorage.DTOs;

public class BookReadDTO
{
    public string Name { get; set; }
    public string Author { get; set; }
    public int PublishYear { get; set; }
    public decimal Price { get; set; }
    public string Category { get; set; }
}

File: client/src/app/models/book-read-dto.ts
export class BookReadDto {
  public id: string;
  public name: string;
  public author: string;
  public publicationYear: number;
  public price: number;
  public category: string;

  constructor(
    id: string,
    name: string,
    author: string,
    publicationYear: number,
    price: number,
    category: string,
  ) {
    this.id = id;
    this.name = name;
    this.author = author;
    this.publicationYear = publicationYear;
    this.price = price;
    this.category = category;
  }
}
